pennylane_lightning/src/tests/Test_DynamicDispatcher.cpp

```diff
@@ -83,7 +83,7 @@ constexpr void testAllGatesForKernelIter(RandomEngine &re,
     if constexpr (idx < static_cast<int>(GateOperation::END)) {
         constexpr auto gate_op = static_cast<GateOperation>(idx);
 
-        for (size_t num_qubits = 3; num_qubits <= max_num_qubits;
+        for (size_t num_qubits = 4; num_qubits <= max_num_qubits;
```

Comment:
Is it possible to change the starting number to the number of qubits that the gate requires i.e. `Util::static_lookup<gate_op>(Constant::gate_wires)`?

Reply:
Fixed it! Thanks :slightly_smiling_face: